tests/jsco4180.py

```diff
@@ -79,21 +79,33 @@ def test_GIVEN_wrong_component_on_device_WHEN_send_new_sp_THEN_state_machine_abo
 
         # Setting an incorrect component on the device will result in the state machine attempting
         # to rerun the pump and reset components after a delay
-        delay = self.ca.get_pv_value("ERROR:DELAY") # delay before state machine reset
-        self._lewis.backdoor_set_on_device("component_A", value-5)
-        self.ca.assert_that_pv_is("COMP:A", value-5, timeout=5)
-        sleep(delay / 2.0)
-
-        # however if we change setpoint, the loop should start again
-        self.ca.set_pv_value("COMP:A:SP", value-10)
-        self.ca.set_pv_value("COMP:B:SP", value+10)
-        sleep(delay / 2.0 + 1.0) # reset should not have happened
-        self.ca.assert_that_pv_is("COMP:A", value-5)
-        sleep(delay / 2.0) # reset should now have happened
-        self.ca.assert_that_pv_is("COMP:A", value-10, timeout=15)
+        initial_delay = self.ca.get_pv_value("ERROR:DELAY")  # delay before state machine reset
+        delay = 30  # Increase delay to avoid race conditions
+        self.ca.set_pv_value("ERROR:DELAY", delay)
```

Comment:
I'm not sure `ERROR:DELAY` is ever actually used. This might be adding an arbitrary sleep?

Reply:
It is used internally to the jasco ioc by it's state machine. We want it to delay for longer otherwise we get races where the state machine in the ioc is faster than the framework